lambdas/service/app.py

```diff
@@ -779,3 +783,69 @@ def self_url(endpoint_path=None):
         endpoint_path = app.current_request.context['path']
     retry_url = f'{protocol}://{base_url}{endpoint_path}'
     return retry_url
+
+
+@app.route('/url', methods=['POST'], cors=True)
+def shorten_query_url():
+    """
+    Take a URL as input and return a (potentially) shortened URL that will redirect to the given URL
+
+    parameters:
+        - name: url
+          in: body
+          type: string
+          description: URL to shorten
+
+    :return: A 200 response with JSON body containing the shortened URL:
+
+    ```
+    {
+        "url": "http://url.data.humancellatlas.org/ABC"
```

Comment:
```suggestion
        "url": "http://url.data.humancellatlas.org/b3N"
```

So it doesn't look like hex.